Source/WebCore/rendering/mathml/RenderMathMLRow.cpp

```diff
@@ -78,6 +78,8 @@ static RenderMathMLOperator* toVerticalStretchyOperator(RenderBox* box)
 
 void RenderMathMLRow::stretchVerticalOperatorsAndLayoutChildren()
 {
+    handlePositionedAndFloatingChildren();
+
     // First calculate stretch ascent and descent.
     LayoutUnit stretchAscent, stretchDescent;
     for (auto* child = firstChildBox(); child; child = child->nextSiblingBox()) {
```

Comment:
Why aren't you browsing only in-flow children in this loop and in the loop below? and removing the `isOutOfFlowPositioned()` checks?

Reply:
Oops, yep